client/data/types.d.ts

```diff
@@ -11,4 +11,5 @@ export interface State {
 	capital?: CapitalState;
 	paymentIntents?: PaymentIntentsState;
 	files?: FilesState;
```

Comment:
I would encourage us to [avoid `any`](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/general-conventions.md#avoid-any) here and type this as accurately as possible since we know/control the shapre of this state.

```suggestion
	paymentActivity?: PaymentActivityState;
```